Jenkinsfile

```diff
@@ -24,6 +24,8 @@ pipeline {
         INTERCOM_PROD_APP_ID = credentials('INTERCOM_PROD_APP_ID')
         MOONPAY_STAGING_API_KEY = credentials('MOONPAY_STAGING_API_KEY')
         MOONPAY_PROD_API_KEY = credentials('MOONPAY_PROD_API_KEY')
+	MOONPAY_STAGING_SECRET_KEY = credentials('MOONPAY_STAGING_SECRET_KEY')
```

Comment:
what is a secret key? What's the difference btw api and secret key?

Reply:
https://docs.moonpay.com/moonpay/developer-resources/api/server-side-apis/authentication

Reply:
I don't understand why we need both